packages/mock-account-provider/app/lib/requesters.ts

```diff
@@ -3,7 +3,7 @@ import type {
   CreatePeerMutationResponse,
   LiquidityMutationResponse,
   CreatePaymentPointerMutationResponse
-} from '../generated/graphql'
+} from '../../generated/graphql'
```

Comment:
You should check the parse_config.ts file in case the default location of the seed file needs to change, based on this I think it might.

Reply:
Yeah I changed it to `./` on Cairin's recommendation, which seems to resolve when I run it locally with pnpm build and run